packages/ts-morph/src/ProjectContext.ts

```diff
@@ -29,6 +29,7 @@ export class ProjectContext {
     private readonly _compilerOptions: CompilerOptionsContainer;
     private readonly _customTypeChecker: TypeChecker | undefined;
     private readonly _project: Project | undefined;
+    private _projectVersion: number  = 0;
```

Comment:
Perhaps since this only relates to the language service, let's move this over to `LanguageService`.